example/cypress/plugins/index.js

```diff
@@ -1,9 +1,11 @@
 const { install, ensureRequiredBrowserFlags } = require('../../../dist');
 
-module.exports = (on, config) => {
-  install(on, config);
+module.exports = (on) => {
+  install(on);
 
-  on('before:browser:launch', (browser = {}, args) =>
-    ensureRequiredBrowserFlags(browser, args)
-  );
+  on('before:browser:launch', (browser = {}, args) => {
+    args = ensureRequiredBrowserFlags(browser, args);
+    args.push('--headless');
```

Comment:
Is that intentional? I'm asking since it was removed in #21

Reply:
It's was removed from plugging. It's just example. Anyway, cypress doesn't support headless node yet.